### ISSUE TITLE
[Blocked by #113] Author affiliations

### ISSUE THREAD

**Issue by lwrubel** [sul-dlss/hungry-hungry-hippo#329]
Add affiliations fields for person authors. See Figma design: https://www.figma.com/design/g46Nfl5EHG7DfwqupK3G4X/H2-2024?node-id=1116-3660&t=3tUKuazmsH4Uck7O-4

**Comment by justinlittman** [comment]
For developer consideration: whether to use a local copy of ROR DB or use ROR API.

**Comment by justinlittman** [comment]
@amyehodge We need mappings to cocina.

**Comment by justinlittman** [comment]
Note to developer: Feel free to split this up as necessary.

**Comment by amyehodge** [comment]
<img width="1379" height="302" alt="Image" src="https://github.com/user-attachments/assets/f4add598-17a1-4e5a-8ccc-08ba44807a41" />

https://www.figma.com/design/g46Nfl5EHG7DfwqupK3G4X/H3-2024?node-id=1114-2853&t=esXU87bcvEWuYxcl-4
